app/scripts/views/base.js

```diff
@@ -329,31 +333,59 @@ define(function (require, exports, module) {
     },
 
     /**
-     * Translate a string
+     * Translate a string, output will be HTML escaped.
      *
      * @param {String} text - string to translate
+     * @param {Object} [context] - interpolation context, defaults to
+     * this.getContext();
      * @returns {String}
      */
-    translate (text) {
-      return this.translator.get(text, this.getContext());
+    translate (text, context = this.getContext()) {
+      return _.escape(this.translator.get(text, context));
     },
 
     /**
-     * Create a function that can be used by Mustache
-     * to translate a string. Useful for translate a string
-     * for use in the template, which itself depends on
-     * this.getContext(). This function avoids
-     * infinite recursion.
+     * Translate a string, do not escape the output.
+     * This should rarely be used, prefer `translate`
+     *
+     * ** WARNING ** DOES NOT HTML ESCAPE
+     *
+     * @param {String} text - string to translate
+     * @param {Object} [context] - interpolation context, defaults to
+     * this.getContext();
+     * @returns {String}
+     */
+    unsafeTranslate (text, context = this.getContext()) {
+      if (Strings.hasUnsafeVariables(text)) {
+        throw new Error(`Unsafe string to translate: ${text}`);
```

Comment:
Instead of a hard exception, maybe just log these for a train to see if we missed any?